templates/edit.html

```diff
@@ -1,20 +1,60 @@
+        <div class="modal-header">
+          <h5 class="modal-title" id="staticBackdropLabel">Warning! You are about to delete an entry!</h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+            <p>Deleting this entry will delete all associated media, including imagery, feedback, artist, and all details of the piece.</p>
+            <p><b>This cannot be undone.</b></p>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-primary" data-bs-dismiss="modal">Close</button>
+          <form action="/delete/{{ muralDetails['id'] }}" method="post">
+          <button type="submit" class="btn btn-danger">Delete Mural</button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  
 
 <div class="container">
-    <h1>{{ muralDetails["title"] }} </h1>
+    <form action="/edittitle/{{ muralDetails['id'] }}" method="post">
+            <input value="{{ muralDetails['title'] }}" type="text" class="form-control" id="title" name="title" required>
+            <button type="submit" class="btn btn-primary">Edit Title</button>
+    </form>
     <div class="row">
         <div class="col-8 mb-3">
             {% for image in muralDetails["images"]: %}
             <img style="width:100%;" src="{{ muralDetails['images'][loop.index-1]['imgurl'] }}">
+            <form action="/makethumbnail?muralid={{ muralDetails['id'] }}&imageid={{ muralDetails['images'][loop.index-1]['id'] }}" method="post">
+                <button type="submit" class="btn btn-primary">Make this the thumbnail</button>
+            </form>
             <form action="/editimage/{{ muralDetails['images'][loop.index-1]['id'] }}" method="post">
                 <div class="mb-3">
                     <label for="caption" class="form-label">Caption</label>
-                    <input value="{{ muralDetails['images'][loop.index-1]['caption'] }}" type="text" class="form-control" id="caption" name="caption" required>
+                    <p class="thin">Write a caption for this image. This should not be a description, as that is covered by the alt text and description fields for the mural. Also include attribution here.</p>
+                    <textarea type="text" class="form-control" id="caption" name="caption">{{ muralDetails['images'][loop.index-1]['caption'] }}</textarea>
                 </div>
                 <div class="mb-3">
                     <label for="alttext" class="form-label">Alt Text</label>
-                    <input value="{{ muralDetails['images'][loop.index-1]['alttext'] }}" type="text" class="form-control" id="alttext" name="alttext" required>
+                    <p class="thin">Write a descriptive caption of this image in particular. Refer to <a href="https://developers.google.com/tech-writing/accessibility/self-study/write-alt-text#write-useful-text">this article</a> on writing good alt text.</p>
```

Comment:
Maybe reword to "a description of this image" to avoid using "caption" since that refers to the previous field?